modules/ROOT/pages/platform/cloud-providers.adoc

```diff
@@ -1,42 +1,86 @@
 [[aura-cloud-providers]]
 = Aura with cloud provider marketplaces
 
-Using Neo4j Aura via a cloud provider marketplace enables you to provision and manage Neo4j Aura databases from your chosen cloud provider console, while taking advantage of any integrated billing.
+== Aura Enterprise
 
-The currently available cloud providers are:
+label:AuraDB-Enterprise[]
+label:AuraDS-Enterprise[]
+
+You can purchase Neo4j Aura Enterprise via private offer through the following cloud provider marketplaces:
 
-* <<aura-getting-started-cloud-providers-GCP, Google Cloud Platform (GCP)>>
-* <<aura-getting-started-cloud-providers-AWS, Amazon Web Services (AWS)>>
-* <<aura-getting-started-cloud-providers-AZURE, Microsoft Azure (Azure)>>
+* Amazon Web Services (AWS)
+* Microsoft Azure (Azure)
+* Google Cloud Platform (GCP)
 
-[[aura-getting-started-cloud-providers-GCP]]
-== Aura with GCP
+To discuss private offers, contact marketplace-sales@neo4j.com.
+
+== Aura Professional
 
 label:AuraDB-Professional[]
 label:AuraDS-Professional[]
 
+You can purchase Neo4j Aura Professional on a pay-as-you-go basis through the following cloud provider marketplaces:
+
+* Amazon Web Services (AWS)
+* Google Cloud Platform (GCP)
+
+Purchasing Neo4j Aura Professional through a cloud provider marketplace gives you access to integrated billing and usage reporting in your chosen cloud provider's console.
+
+=== AWS
+
+[discrete]
+==== 1. Purchase the service
+
+To get started, visit the https://aws.amazon.com/marketplace/pp/prodview-2t3o7mnw5ypee[Neo4j Aura Professional AWS marketplace page] and select *View Purchase options*.
```

Comment:
I think "marketplace" should we capitalised here (and in the following when referred to AWS).